source/features/vertical-front-matter.tsx

```diff
@@ -0,0 +1,43 @@
+import './vertical-front-matter.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
```

Comment:
Init goes on the bottom of the file

Reply:
There is only one function (`init`), I don't get you.

Reply:
Sorry my bad